### PR TITLE
Add entire content of Scholarly Intro and created ToC, WC, and footnotes

### PR DESCRIPTION
**Pull request by grenerad**
I copied the general "import" and ToC script from the general intro, and added the entire content for the Scholarly Introduction page.

### CODE REVIEW

**Comment by grenerad** [comment]
I felt comfortable enough working in mdx to try and create the Scholarly Introduction page, using the already established pages to hopefully generate this page. We will want to add in images later, but for now I hope this gets the page established without too many errors to fix.

File: src/pages/introduction/scholarly.mdx [hunk 1]
```diff
@@ -4,4 +4,90 @@ title: "SCHOLARLY INTRODUCTION: Dickens’s Working Notes"
 description: "Charles Dickens’s Working Notes are ..."
 ---
 
-TODO
+import ImageWithCaption from "/src/components/ImageWithCaption.astro";
+import TOC from "/src/components/TOC.astro";
+
+{/* prettier-ignore */}
+<TOC>
+ - [Rethinking Dickens's Serial Form with the DDNP](#rethinking-dickens's-serial-form-with-the-ddnp)
```
Comment (csbailey5t) [inline src/pages/introduction/scholarly.mdx:12]: You'll need to remove the apostrophe from the # tag link. When Astro creates the ID for the header, it simply lowercases, strips punctuation, and concatenates with hyphens.

File: src/pages/introduction/scholarly.mdx [hunk 2]
```diff
@@ -4,4 +4,90 @@ title: "SCHOLARLY INTRODUCTION: Dickens’s Working Notes"
...
+
+## Pattern and Process: Dickens as Story-Weaver
+
+The tension between finished form and compositional process was central to Dickens’s own experience of his serial novels. In his prefaces, which he wrote immediately after finishing a novel, he often used the metaphor of weaving to name the conflicting pressures of the “general purpose and design” and “the temptation of the current monthly number” (_Martin Chuzzlewit_ 5). In the preface to _Little Dorrit_ (1855-57), for instance, he made this plea to his readers:
+
+_[A]s it is not unreasonable to suppose that I may have held [the work’s] threads with a more continuous attention than anyone else can have given them during its desultory publication, it is not unreasonable to ask that the weaving may be looked at in its completed state, and with the pattern finished._ (xxi)
```
Comment (csbailey5t) [inline src/pages/introduction/scholarly.mdx:39]: Markdown has markup for block quotes that you might want to try. Put a ">" at the beginning of the first line of the quote to check it out.

**Review by csbailey5t — CHANGES_REQUESTED** [review]
This is great @grenerad. I left a couple -- one thing to get a TOC link working, and one suggestion for formatting for block quotes.

File: src/pages/introduction/scholarly.mdx (hunk 2 above)
Comment (grenerad) [reply src/pages/introduction/scholarly.mdx:39]: This is great--thank you @csbailey5t. These were two questions I had as I was doing it, so I will make these changes and merge. Anna and I can discuss the block quotations as a matter of style

**Pull request merged**